[2023-09-02 13:16:57,405] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: entregable3.extract_and_transform_data manual__2023-09-02T13:16:56.339144+00:00 [queued]>
[2023-09-02 13:16:57,417] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: entregable3.extract_and_transform_data manual__2023-09-02T13:16:56.339144+00:00 [queued]>
[2023-09-02 13:16:57,418] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-09-02 13:16:57,418] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-09-02 13:16:57,419] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-09-02 13:16:57,434] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extract_and_transform_data> on 2023-09-02 13:16:56.339144+00:00
[2023-09-02 13:16:57,438] {standard_task_runner.py:52} INFO - Started process 219 to run task
[2023-09-02 13:16:57,442] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'entregable3', 'extract_and_transform_data', 'manual__2023-09-02T13:16:56.339144+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/entregable3.py', '--cfg-path', '/tmp/tmp8yhhqvny', '--error-file', '/tmp/tmpdjt1u1ng']
[2023-09-02 13:16:57,444] {standard_task_runner.py:80} INFO - Job 139: Subtask extract_and_transform_data
[2023-09-02 13:16:57,516] {task_command.py:371} INFO - Running <TaskInstance: entregable3.extract_and_transform_data manual__2023-09-02T13:16:56.339144+00:00 [running]> on host eebeded14cb4
[2023-09-02 13:16:57,631] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Yamil
AIRFLOW_CTX_DAG_ID=entregable3
AIRFLOW_CTX_TASK_ID=extract_and_transform_data
AIRFLOW_CTX_EXECUTION_DATE=2023-09-02T13:16:56.339144+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-09-02T13:16:56.339144+00:00
[2023-09-02 13:17:05,562] {logging_mixin.py:115} INFO - No hay nuevos datos para cargar en la tabla.
[2023-09-02 13:17:05,563] {python.py:173} INFO - Done. Returned value was: None
[2023-09-02 13:17:05,578] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=entregable3, task_id=extract_and_transform_data, execution_date=20230902T131656, start_date=20230902T131657, end_date=20230902T131705
[2023-09-02 13:17:05,623] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-09-02 13:17:05,651] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
